src/test/java/org/terracotta/statistics/simulation/latency/Percentile.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * All content copyright Terracotta, Inc., unless otherwise indicated.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terracotta.statistics.simulation.latency;
+
+import org.apache.commons.math3.stat.descriptive.DescriptiveStatistics;
+import org.terracotta.statistics.derived.latency.LatencyHistogramStatistic;
+
+import java.text.NumberFormat;
+import java.util.function.DoubleUnaryOperator;
+
+/**
+ * @author Mathieu Carbou
+ */
+public class Percentile {
```

Comment:
How is this a percentile?  Isn't this just a closed range or interval?  The percentileness is a feature of how it's used.

Reply:
Yes I'll rename the class. The name is wrong here.